pkg/controller/operands/ssp.go

```diff
@@ -168,13 +168,18 @@ func (h *sspHooks) updateCr(req *common.HcoRequest, client client.Client, exists
 
 func NewSSP(hc *hcov1beta1.HyperConverged, opts ...string) *sspv1beta1.SSP {
```

Comment:
This is a good test, but please also add test/test case in pkg/controller/hyperconverged/hyperconverged_controller_test.go